```
/agent/_work/1/s/src/SamplesApp/SamplesApp.Skia.netcoremobile/SamplesApp.Skia.netcoremobile.csproj : error NU1102: Unable to find package Microsoft.NETCore.App.Runtime.Mono.linux-x64 with version (= 10.0.5)
/agent/_work/1/s/src/SamplesApp/SamplesApp.Skia.netcoremobile/SamplesApp.Skia.netcoremobile.csproj : error NU1102:   - Found 140 version(s) in NuGet official package source [ Nearest version: 9.0.0-preview.7.24405.7 ]
/agent/_work/1/s/src/SamplesApp/SamplesApp.Skia.netcoremobile/SamplesApp.Skia.netcoremobile.csproj : error NU1102:   - Found 0 version(s) in Solution Packages

```

@MartinZikmund Looks unrelated to my change. Is that a known build error currently? Is it related to https://github.com/unoplatform/uno/issues/23010?

--- a/src/Directory.Build.targets
+++ b/src/Directory.Build.targets
@@ -100,7 +100,7 @@
 		<PackageReference Update="Uno.Wasm.WebSockets" Version="1.1.0" />
 		<PackageReference Update="Microsoft.TypeScript.MSBuild" Version="4.3.5" />
 		<PackageReference Update="NUnit" Version="3.14.0" />
-		<PackageReference Update="NUnit3TestAdapter" Version="6.0.0" />
+		<PackageReference Update="NUnit3TestAdapter" Version="6.2.0" />
 		<PackageReference Update="NunitXml.TestLogger" Version="3.1.15" />
 		<PackageReference Update="Microsoft.Testing.Extensions.TrxReport" Version="$(MicrosoftTestingPlatformVersion)" />
 		<PackageReference Update="Uno.UITest" Version="1.1.0-dev.70" />

